presets/test/manifests/falcon-7b-with-adapter/falcon-7b.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: falcon-7b
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: falcon
+  template:
+    metadata:
+      labels:
+        app: falcon
+    spec:
+      volumes:
+      - name: shared
+        emptyDir: {}
+      initContainers:
+      - name: init-container-2
+        image: aimodelsregistrytest.azurecr.io/adapter-output-ubuntu:0.0.1
```

Comment:
You cannot check in this url because it is a private ACR url. You can leave the image url empty for now.